pkg/app/app.go

```diff
@@ -24,12 +24,21 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"net/http"
+	"net"
 	"os"
+	"strings"
 	"time"
 
 	kapierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
Nit: Can you organize this so that imports with a name are together and those without a name are together.

```go
kapierrors "k8s.io/apimachinery/pkg/api/errors"
kubeversiontypes "k8s.io/apimachinery/pkg/version"
...

"k8s.io/apimachinery/pkg/runtime/serializer"
"k8s.io/apimachinery/pkg/util/wait"
...

Reply:
go imports organize the imports like this I would prefer to continue to use go imports.

Does anyone else have any thoughts?

Reply:
It's the style we've been using. I don't mind changing it.

Reply:
we could name them all and goimports will be fine. The problem with changing it is that @shawn-hurley and myself will flip it back when we save the file again. Not sure if there's a way to configure goimports. And I'd hate to stop using it since it does so much for me automatically.